virttest/qemu_virtio_port.py

```diff
@@ -98,7 +99,11 @@ def open(self):     # @ReservedAssignment
         attempt = 11
         while attempt > 0:
             try:
-                self.sock = socket.socket(socket.AF_UNIX,
+                if self.port_type == 'unix_socket':
+                    flag = socket.AF_UNIX
+                elif self.port_type in ('tcp_socket', 'udp'):
+                    flag = socket.AF_INET
```

Comment:
No IPv6 support ?

Reply:
@xutian No such requirement currently, and seems will not have it in very long time, since have confirmed with devel before that we don't need test ipv6 for serial device.